superset-frontend/src/views/dashboardList/DashboardList.tsx

```diff
@@ -45,23 +45,24 @@ interface Props {
 }
 
 interface State {
-  dashboards: any[];
+  bulkSelectEnabled: boolean;
   dashboardCount: number;
-  loading: boolean;
+  dashboards: any[];
```

Comment:
Not sure if it's worth the effort, but if instead of `dashboard____ ` keys, these could be made more generic, e.g. `itemCount`, and then `ListView` could export a shared/reusable interface?

Reply:
that's an interesting idea.

Reply:
DatasetList got converted into a functional component with hooks, so the shared interface makes less sense. Shared hooks on the other hand.... 🤔